app/src/main/java/com/example/memeapp/presentation/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.memeapp.presentation.mvp.view
```

Comment:
please move this class to a base package inside the view package.

com.example.memeapp.presentation.mvp.view.base